### ISSUE TITLE
marketing

### ISSUE THREAD

**Issue by danschumann** [satyr/coco#241]
Hey all, I'm glad to see that there is the same spark that died in coffeescript reignited here.  Originally I'm sure they meant to continually reinvent conventions as to what is most efficient and readable, but since I've been to coffeescript, I've seen good, simple, obvious suggestions get shot down **without** explanation.  That sort of unreasonableness has lead me here, and if I have to I'll start reading the code, but the average user will probably expect better and easier to work with docs and api explanation.

Is there a way to `var coco = require('coco'); coco.compile(...)` ?  Probably, right?  Where's the docs?

Most of the doc focus is on what's different.. but what's the same?  Shouldn't this feel like a stand alone repo that can explain itself now that it's maturing?   I know it's a free product but I feel like quite a bit of benefit could be done from quite a small amount of marketing.  After all, I'm sure quite a bit of work goes into the code, so why not show it off?

**Comment by Delapouite** [comment]
@danschumann if you're looking for a more complete documentation, you may be interested by LiveScript which is an offspring of coco (and CoffeeScript obviously) : http://livescript.net

**Comment by vendethiel** [comment]
> I've seen good, simple, obvious suggestions get shot down without explanation

Where?

**Comment by vendethiel** [comment]
Right. Refusing to back up your claims is definitely what fixes issues one encounters.

I'm not saying "you're lying, prove your point", I'm asking you where we could've done beter.

> I am just about done using
> coffeescript at all.

I don't get anything if more people use CoffeeScript. I havn't used it myself for almost two years, modulo stuff I do to to help on IRC. We're just volunteers. Sometimes we don't answer like we should, and that doesn't mean we don't want to try and fix it

**Comment by michaelficarra** [comment]
@danschumann: As @Nami-Doc says, we're just volunteers legitimately trying to help people in their use of compile-to-JS languages. If we don't know what has been bothering you, we can't fix it. I suggest you be more patient in the future.

**Comment by vendethiel** [comment]
@danschumann I'm sorry that's how you feel. But again, I can't change anything if I don't understand what's wrong. Every issue I close on coffee gets an explanation, usually a link to a duplicated ticket with explanations in that. Don't take it personally or think I'm dismissive, "sarcastic", etc.
